conf/wmi.conf.example

```diff
@@ -64,13 +65,49 @@ value = explorer.exe
 action = allow
 EOT
 
+# Detect if FortiClient.exe is running, if yes the Security Event is triggered
```

Comment:
What about if we flip this logic around? I guess we want the security event to be triggered if the FortiClient.exe is NOT running.